schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,623 @@
+        "type": "array",
+        "items": [
+          {
+            "type": "object",
+            "properties": {
+              "title": {
+                "type": "string",
+                "minLength": 1
+              },
+              "language": {
+                "const": "en"
+              }
+            },
+            "additionalProperties": false
+          }
+        ],
+        "additionalItems": {
+          "type": "object",
+          "properties": {
+            "title": {
+              "type": "string",
+              "minLength": 1
+            },
+            "language": {
+               "allOf": [
+                 {"$ref": "#/definitions/language"},
+                 {"not": {"enum": ["en"]}}
+             ]
+            }
+          },
+          "additionalProperties": false
+        }
+      },
+      "author_link": {
+        "tile": "Author link",
+        "description": "Author-provided link to related resource.",
+        "type": "object",
+        "properties": {
+          "uri": {
+            "type": "string",
+            "format": "uri"
+          },
+          "display_text": {
+            "description": "Descriptive text to associate with the URI.",
+            "type": "string",
+            "minLength": 1
+          },
+          "category": {
+            "title": "Link category",
+            "description": "The type of resource the link resolves to.",
+            "type": "string",
+            "$comment": "TODO: refine this enumeration",
+            "enum": ["related article", "slides", "notes", "software", "dataset", "simulation", "video", "review", "news item", "personal website", "blog entry", "other"]
+          }
+        },
+        "required": [
+          "uri",
+          "category"
+        ]
+      },
+      "unversioned_identifier": {
+        "title": "arXiv canonical identifier",
+        "type": "string",
+        "pattern": "^(([0-9]{4}\\.[0-9]{4,5})|([a-z\\-]+\\/[0-9]{2}[01][0-9]{4}))$"
+      },
+      "versioned_identifier": {
+        "title": "arXiv canonical identifier with version affix",
+        "type": "string",
+        "pattern": "^(([0-9]{4}\\.[0-9]{4,5})|([a-z\\-]+\\/[0-9]{2}[01][0-9]{4}))v[0-9]+$"
+      },
+      "version": {
+
+        "type": "object",
+        "properties": {
+          "content_types": {
+            "title": "List of content types",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/content_type"
+            },
+            "minItems": 1,
+            "uniqueItems": true
+          },
+          "submitter": {
+            "title": "Submitter",
+            "description": "Information about the submitter and possible proxy for the submitter.",
+            "type": "object",
+            "properties": {
+              "submitter": {
+                "$ref": "#/definitions/person"
+              },
+              "proxy_name": {
```

Comment:
Add ORCID to proxy?

Reply:
Potentially, though a proxy is usually a non-author submitting on behalf of one.

Reply:
https://arxiv-org.atlassian.net/browse/ARXIVNG-3523?jql=text%20~%20DOI%20AND%20project%20%3D%20arXiv-NG%20ORDER%20BY%20status%20DESC